app/score_calculator/yaku_check/blocks_yaku_checker.py

```diff
@@ -133,6 +158,45 @@ def is_two_terminal_chows(self) -> bool:
             and abs(self.blocks[0].tile - self.blocks[1].tile) == self.TERMINAL_GAP
         )
 
+    # four blocks checker
+    @property
+    def is_big_four_winds(self) -> bool:
+        return self.validate_all_conditions(Block.is_wind, Block.is_pung)
+
+    @property
+    def is_little_four_winds(self) -> bool:
+        return (
+            self.validate_all_conditions(Block.is_wind)
+            and len(
+                list(
+                    filter(lambda x: x.type == BlockType.PAIR, self.blocks),
+                ),
+            )
+            == 1

```

Comment:
이 경우는 갯수가 작아서 큰 상관없겠지만, 중간 list를 생성하는 게 일반적으로 좋지 못합니다.
일반적인 practice는 다음과 같습니다.
```suggestion
            and sum(1 for block in self.blocks if block.type == BlockType.PAIR) == 1

```

Reply:
오 좋네요 감사합니다